packages/langium/src/documentation/jsdoc.ts

```diff
@@ -562,6 +567,10 @@ class JSDocTagImpl implements JSDocTag {
     }
 
     toMarkdown(options?: JSDocRenderOptions): string {
+        return options?.renderTag?.(this) ?? this.toMarkdownDefault(options);
```

Comment:
The comment above says the return value is `undefined` if the tag is not valid, but here it's treated the same as if `renderTag` is not defined. Shall we handle the two cases differently (`renderTag` undefined, `renderTag` returning undefined)?

Reply:
The comment

```
/**
 * Custom tag rendering function.
 * Return a markdown formatted tag or `undefined` if the tag is not valid.
 */
```

is misleading indeed. How about this instead:

```
/**
 * Custom tag rendering function.
 * Return a markdown formatted tag or `undefined` to fall back to the default rendering.
 */
```

Reply:
Ok 👍

Reply:
I've changed the comment accordingly.